rust/windows-client/src-tauri/src/client/gui.rs

```diff
@@ -293,14 +331,10 @@ impl connlib_client_shared::Callbacks for CallbackHandler {
         Ok(())
     }
 
-    fn on_update_resources(
-        &self,
-        resources: Vec<connlib_client_shared::ResourceDescription>,
-    ) -> Result<(), Self::Error> {
-        tracing::trace!("on_update_resources");
-        // TODO: Better error handling?
-        self.ctlr_tx
-            .try_send(ControllerRequest::UpdateResources(resources))?;
+    fn on_update_resources(&self, resources: Vec<ResourceDescription>) -> Result<(), Self::Error> {
+        tracing::debug!("connlib sees {} resources", resources.len());
```

Comment:
feels like too much logging, we should already see the resource list

Reply:
I was worried about the odd `Notify` trick I'm using causing the GUI to miss an update from connlib and lag behind. It looks okay in the one manual test I did, so I could drop it to trace or get rid of it I guess